src/main/java/org/galatea/starter/service/IexCloudApiClient.java

```diff
@@ -0,0 +1,16 @@
+package org.galatea.starter.service;
+
+import java.util.List;
+import org.galatea.starter.domain.IexHistoricalPrices;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@FeignClient(name = "IEXCLOUD", url = "${spring.rest.iexCloudApiBasePath}")
+public interface IexCloudApiClient {
+
+  @GetMapping("/stock/{symbol}/chart/{range}/{date}?token=${spring.rest.api_Token}")
```

Comment:
generally speaking, underscores aren't used in fields like this in java. Change to camelCase